login-form/style.css

```diff
@@ -0,0 +1,69 @@
+@import url("https://fonts.googleapis.com/css2?family=Josefin+Sans&display=swap");
+* {
+  margin: 0;
+  font-family: "Josefin Sans", sans-serif;
+  font-size: 20px;
+  margin: 0;
+}
+
+.flex {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.container {
+  flex-direction: column;
+  background: url("https://cdn.pixabay.com/photo/2015/12/01/20/28/road-1072823_960_720.jpg");
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: 100vh;
+}
+
+.login-form {
+  flex-direction: column;
+  min-width: 340px;
+  height: 400px;
+  background: rgba(255, 255, 255, 0.775);
+  box-shadow: 1px 2px 5px;
+  border-radius: 10px;
+  min-height: 300px;
+  box-shadow: 10px 10px 60px;
+}
+
+.element {
+  padding: 10px;
+}
+
+.login-btn {
+  margin-top: 20px;
+  background: rgb(241, 183, 7);
+  height: 35px;
+  border-radius: 10px;
+  box-decoration-break: slice;
+  box-shadow: 2px 2px 10px rgb(255, 183, 0);
+  color: rgb(255, 255, 255);
+  font-weight: 600;
+}
+
+.uname,
+.pwd {
+  height: 15px;
+  font-size: 14px;
+  margin-top: 5px;
+  color: red;
+}
+
+.username,
+.password {
+  font-size: 16px;
+  margin: 4px;
+  min-width: 240px;
+}
+
+.pwdRules {
+  margin: 20px;
+  height: 60px;
+  width: 200px;
+}
```

Comment:
Revamp UI design to resemble preview image.

Reply:
done